### ISSUE TITLE
Cookie consent banner is not keyboard accessible

### ISSUE THREAD

**Issue by cloudrac3r** [hacks-guide/Guide_Landing#32]
The "got it" button in the cookie consent banner cannot be activated with the keyboard, which is quite bad for accessibility.
